storage/statedb/cache_redis.go

```diff
@@ -18,13 +18,15 @@ package statedb
 
 import (
 	"errors"
+	"runtime"
 	"time"
 
 	"github.com/go-redis/redis/v7"
 	"github.com/klaytn/klaytn/common/hexutil"
 )
 
 const (
+	redisSetItemChannelSize       = 1024
```

Comment:
1. Do you think this is enough size to handle all items?
  Last week, 1318 WCU was used in DynamoDB. 
2. Does channel size with power of 2 perform better?

Reply:
1. In the test, the size was enough to execute `debug.StartWarmUp()` without any errors. Redis could handle write commands faster than DynamoDB in more cases. Then, the channel will not be fulled.
2. I don't know exactly. Even if it does, the difference is probably not quite big. It just took the number thinking that it might help to allocate memory efficiently. However, the difference is probably not quite big also. If you have any other opinions, please let me know.